portfolio/src/main/webapp/script.js

```diff
@@ -29,5 +33,12 @@ function addRandomGreeting() {
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = fact;//greeting;
+  
+
+  const img = document.getElementById('visit-pic');
+  img.src = visit_pics[fact];
+  img.alt = fact + '-image';
```

Comment:
alt text is not a tag, it's about accessibility. Please update this to be something that someone using a screen reader would want to know, like "An image of ${fact}." or something.